server/store/kvstore/kvstore.go

```diff
@@ -0,0 +1,6 @@
+package kvstore
+
+type KVStore interface {
+	// Define your methods here. This package is used to access the KVStore pluginapi methods.
+	GetTemplateData(userID string) (string, error)
```

Comment:
What do we use this for?

Reply:
Nothing currently. It's just part of the starter-template, along with a bunch of other stuff not needed.  I kept this thinking we'll *probably* use the KV store, but I can remove it. 

I think we need some sort of template generator that let's people pick what features they want in their plugin.  Currently the template has a lot of cruft not needed by all plugins. Removing those features makes them hard to discover later.  A menu of features to add would help a lot.